packages/frontend/web-ui/src/app/theme/cornieTheme.ts

```diff
@@ -1,9 +1,18 @@
-import { createTheme } from '@mui/material/styles';
+import { Theme } from '@emotion/react';
+import {
+  PaletteOptions,
+  SimplePaletteColorOptions,
+  createTheme,
+} from '@mui/material/styles';
 
-/** @type {import("@mui/material/styles").PaletteOptions } */
-const corniePalette = {
+interface CorniePaletteOptions extends PaletteOptions {
+  primary?: SimplePaletteColorOptions;
+  secondary?: SimplePaletteColorOptions;
+}
+
+const corniePalette: CorniePaletteOptions = {
   primary: {
-    white: '#fff',
+    //white: '#fff',
```

Comment:
Avoid comment in favor of removing the line

Reply:
Deleted